libpod/container_api.go

```diff
@@ -230,10 +230,9 @@ func (c *Container) Kill(signal uint) error {
 	}
 
 	c.state.StoppedByUser = true
-
 	c.newContainerEvent(events.Kill)
 
-	return c.save()
+	return c.recordExitCode()
```

Comment:
podman kill can be used to send any signal to the container, unless I am missing something `c.recordExitCode()` would block and wait for conmon to exit which seems wrong. Am I missing something?

Reply:
Oh, that is a fair point. I'll check what I can do.

Reply:
I concur. It seems there are still some farts in the current logic. Will continue digging tomorrow.